web/lib/date/Date.php

```diff
@@ -0,0 +1,1465 @@
+<?php
```

Comment:
I had done some of that previously by breaking them out into managed Composer packages, but that requires an independent git repo for each one. When we were working on Text_Wiki that ended up being a bother and would generally slow down things since we moved to a monorepo.

We will also need to namespace every call to one of these libraries when we do that, which won't be bad for most of them but for Text_Wiki will be a large-ish task.